cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/etl/transform/ScriptLookupTable.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright © 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package co.cask.cdap.etl.transform;
+
+import sun.org.mozilla.javascript.internal.NativeObject;
```

Comment:
Currently using `NativeObject` to construct a map that Rhino/JavaScript can use. Maybe there's some alternative, but I'm not aware of it.

A better solution may be to depend on the Rhino maven artifact in cdap-etl-lib instead of assuming it exists.